cost-analyzer/values.yaml

```diff
@@ -512,6 +512,26 @@ networkCosts:
       #   zone: "us-east1-c"
       #   ips:
       #     - "10.0.0.0/24"
+    services:
+      # google-cloud-services: when set to true, enables labeling traffic metrics with google cloud 
+      # service endpoints 
+      google-cloud-services: false
+      # amazon-web-services: when set to true, enables labeling traffic metrics with amazon web service 
+      # endpoints.
+      amazon-web-services: false
```

Comment:
nit: but i may got aws-cloud-services if you keep this flag?

Reply:
"Amazon Web Services Cloud Services"?

I was going off the branding used by the providers.

Reply:
It's also important to note here that currently, this configuration is a direct passthrough to the network-costs, so we don't require any custom helm mapping. The network costs image for v16.0 has already been cut, which expects this configuration. Changing an option in the helm chart would require a special config mapping which just wouldn't be worth the effort.

Reply:
Works for me, but I would highlight that that specific language is actually used... 

![image](https://user-images.githubusercontent.com/298359/150621889-e8f6c2c9-3b3d-45bd-a6c1-0b1786770d44.png)